docs/scenarios/parametric.md

```diff
@@ -112,7 +112,19 @@ TEST_LIBRARY=java ./run.sh test_span_sampling.py::test_single_rule_match_span_sa
 
 #### PHP
 
-If you are seeing DNS resolution issues when running the tests locally, add the following config to the Docker daemon:
+##### To run with a custom build
+
+- Download `datadog-setup.php` in `build packages/package extension` job on CI
```

Comment:
Very good point. Plus it's actually the name of the workflow. 
Next time don't hesitate to add a suggestion, it's easier I just have to accept :)
Thank you
```suggestion
- Download `datadog-setup.php` in `build_packages/package extension` job on CI
```